Source/Core/Core/PowerPC/Jit64/Jit_Branch.cpp

```diff
@@ -90,11 +91,12 @@ void Jit64::bx(UGeckoInstruction inst)
 #endif
   if (destination == js.compilerPC)
   {
-    // PanicAlert("Idle loop detected at %08x", destination);
-    // CALL(ProtectFunction(&CoreTiming::Idle, 0));
-    // JMP(Asm::testExceptions, true);
-    // make idle loops go faster
-    js.downcountAmount += 8;
+    ABI_PushRegistersAndAdjustStack({}, 0);
+    ABI_CallFunction(CoreTiming::Idle);
+    ABI_PopRegistersAndAdjustStack({}, 0);
+    MOV(32, PPCSTATE(pc), Imm32(destination));
```

Comment:
I think so, as the Jit64 has no immediate version of WriteExceptionExit. But you're right, not supporting this is bad, but out of scope of this PR.